proto/server.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
```

Comment:
`go generate` honors `GOPATH`. You can set that for the Makefile environment it'll work out of the box. Having `go generate` supports a very succinct workflow when experiment with proto changes.